scripts/gen_gcov_files.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# This script will parse the serial console log file and create the required
+# gcda files.
+# Usage python3 ${ZEPHYR_BASE}/scripts/gen_gcov_files.py -i console_output.log
+# Add -v for verbose
+
+
+
+import sys
+import argparse
+import os
+import re
+import string
+import hexdump
```

Comment:
This is a new dependency.

Before introducing a new dependency one should evaluate whether the functionality can be covered with the existing dependencies.

AFAICT, we could replace "dehex" with https://stackoverflow.com/a/444005 or something similair to avoid this new dependency.